app/controllers/api/projects_controller.rb

```diff
@@ -2,11 +2,25 @@
 
 class Api::ProjectsController < Api::ApplicationController
   before_action :find_project, except: %i[searchcode dependencies dependencies_bulk updated]
+  before_action :require_api_key, only: %i[update]
+  before_action :require_api_key_for_update, only: %i[update]
+
+  ALLOWED_UPDATE_API_KEY_IDS = [1].freeze
 
   def show
     render(json: @project, show_updated_at: internal_api_key?)
   end
 
+  def update
+    # currently this endpoint only updates the "lifted" attribute. No need for callbacks/validations yet.
+    project_params = params.require(:project).permit(:lifted)
+    @project.update_columns(lifted: project_params[:lifted])
```

Comment:
`update_columns` still uses AR's builtin type-casting here, so anything that looks like a "false" value will get saved as false (or true, otherwise).